core/src/main/java/org/apache/carbondata/core/util/DeleteLoadFolders.java

```diff
@@ -183,16 +186,18 @@ private static boolean checkIfLoadCanBeDeleted(LoadMetadataDetails oneLoad,
         return true;
       }
       long deletionTime = oneLoad.getModificationOrDeletionTimestamp();
-
+      if (isCleanFilesOperation) {
```

Comment:
this option is not required.
Only clean files can do clean data.

Reply:
done